tests/ReactTestApp/ios/ReactTestApp.xcodeproj/xcshareddata/xcschemes/ReactTestApp.xcscheme

```diff
@@ -98,18 +55,11 @@
          <BuildableReference
             BuildableIdentifier = "primary"
             BlueprintIdentifier = "13B07F861A680F5B00A75B9A"
-            BuildableName = "ReactTests.app"
+            BuildableName = "ReactTestApp.app"
             BlueprintName = "ReactTestApp"
-            ReferencedContainer = "container:ReactTests.xcodeproj">
+            ReferencedContainer = "container:ReactTestApp.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
-      <EnvironmentVariables>
-         <EnvironmentVariable
-            key = "OS_ACTIVITY_MODE"
-            value = "disable"
-            isEnabled = "YES">
-         </EnvironmentVariable>
-      </EnvironmentVariables>
```

Comment:
You probably do want this env var. Anything involving networking logs a lot of useless garbage without it.

Reply:
Great tip.